ext/semian/simple_integer.c

```diff
@@ -65,69 +65,77 @@ VALUE semian_simple_integer_increment(int argc, VALUE *argv, VALUE self) {
   // This is definitely the worst API ever.
   // https://silverhammermba.github.io/emberb/c/#parsing-arguments
   VALUE val;
+  semian_simple_integer_t *res;
+  VALUE rb_val;
```

Comment:
Convention is usually to use `retval` for the name here, to clarify that it's the thing that will ultimately be returned.